README.rst

```diff
@@ -25,8 +25,8 @@ Interface* (`MPI <https://www.mpi-forum.org/>`_) standard. It is
 implemented on top of the MPI specification and exposes an API which
 grounds on the standard MPI-2 C++ bindings.
 
-Dependencies
-------------
+Prerequisites
+-------------
```

Comment:
I have to say this is really like babysitting, I see no value added, but now that you've made the change I wouldn't even spend a second to argue 🙂

Reply:
The new word sounds more pretentious... isn't that enough of a good reason for the change?